app/representers/api/auth/feed_representer.rb

```diff
@@ -7,6 +7,10 @@ class Api::Auth::FeedRepresenter < Api::BaseRepresenter
   property :file_name
   property :private
   property :title
+  property :subtitle
+  property :description
+  property :summary
+  property :summary_preview, exec_context: :decorator, if: ->(_o) { represented.summary.blank? }
```

Comment:
This seems to be a Publish display thing?  If you have a blank/nil `summary`, it displays what the sanitized-links-only version of the `description` is.

Reply:
yeah, as I recall, that's how the summary shows up in iTunes/Apple Podcasts, which supports links, and not much else.